_posts/2016-10-15-recurse-center-week-3.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "Week[2] @ The Recurse Center 🌻"
+comments: true
+categories: tech recurse
+---
+
+1/4 of the way through! Time is flying.
+This post is also quite late. I suppose it's a good sign that I've been too busy to write.
+
+# Overall
+
+The week started off on a low, and steadily improved to wonderful levels. 😊
+
+### Emotional 😱
+
+#### **I felt emotionally drained.**
+
+At the start of the week I felt pretty emotionally drained.
+
+My past experiences with making connections/getting involved with communities involved a lot of putting myself out there and stepping out of my comfort zone.
+It's a bit of a numbers game. The more you do these things, the higher your chances of finding those people you click with. At the same time, it can go overboard.
+
+It can become forced.
+It can turn into a lack of boundaries.
+It can have the opposite effect.
+
+When you over invest, you can go bankrupt. Investments need space/time to grow.
+
+#### **I questioned the value of my offers.**
+
+A sticky trap I fall into is self-doubt when my expectations of others don't line up with how things play out in real life.
+I actually have had only positive interactions with people in person. Yet when I was extending offers for events/activities, there was less enthusiasm than expected.
+
+I started wondering if my offers had value. Or whether the quality of these interactions were as good as I thought.
+
+#### **I wondered if RC was wrong for me.**
+
+Another thought was that maybe RC just wasn't compatible with me.
+I actually got rejected the first time I applied. Not even after an interview...right after the application.
+
+Maybe my social goals weren't aligned with this space.
+Maybe RC is more focused on the technical growth.
+Maybe I didn't belong.
+
+#### **I focused on things outside the space.**
+
+I realized there was a lot for me to explore outside of RC. As a result, no evenings were spent 100% at RC last week.
```

Comment:
FWIW I've never stayed after 6:30pm unless it was game night and I sometimes feel bad about it but it's fine to enjoy NYC or have some time off programming after a long day. Some people are here for like 18 hours a day but I just can't do it!